ISICbalance/ViewControllers/BalanceViewController.swift

```diff
@@ -99,44 +105,56 @@ class BalanceViewController: BaseViewController {
     }
     
     fileprivate func setupButtonsStack() {
-        let buttonsStackView = UIStackView()
-        buttonsStackView.distribution = .fillEqually
-        screenStackView.addArrangedSubview(buttonsStackView)
-
         let reloadButton = UIButton()
         reloadButton.setImage(Asset.reloadIcon.image, for: .normal)
         self.reloadButton = reloadButton
-        buttonsStackView.addArrangedSubview(reloadButton)
 
         let accountButton = UIButton()
         accountButton.setImage(Asset.accountIcon.image, for: .normal)
         accountButton.isUserInteractionEnabled = true
         self.accountButton = accountButton
-        buttonsStackView.addArrangedSubview(accountButton)
+
+        let buttonsStackView = UIStackView(arrangedSubviews: [reloadButton, UIView(), accountButton])
+        buttonsStackView.distribution = .fillEqually
+        screenStackView.addArrangedSubview(buttonsStackView)
     }
 
     // MARK: - Bindings
     func setupBindings() {
-        self.balanceLabel.reactive.text <~ viewModel.balance
-        // push accountViewController if there is some error duting balanceAction 
-        viewModel.getBalanceAction.errors.producer.startWithValues { [weak self] error in
-            guard case RequestError.successfulParse = error else {
-                // TODO: nevolat handle odjinud než z buttonu
-                self?.accountBtnHandle()
-                return
+        self.balanceLabel.reactive.text <~ viewModel.balance.map { $0.asLocalCurrency() }
```

Comment:
máš tu navíc `self`, snaž se tlačit veškerý stav do viewModelů a v controllerech mít `property <~ viewModel.property` bez těch `map`ů. `asLocalCurrency` je věc, která má být na viewModelu